src/miner_restart_sup.erl

```diff
@@ -104,8 +104,8 @@ init(_Opts) ->
                 application:set_env(blockchain, poc_mgr_mod, miner_poc_mgr),
                 application:set_env(sibyl, poc_mgr_mod, miner_poc_mgr),
                 application:set_env(sibyl, poc_report_handler, miner_poc_report_handler),
-                [PocMgrTab1, PocMgrTab2] = miner_poc_mgr:make_ets_table(),
-                POCMgrOpts = #{tab1 => PocMgrTab1, tab2 => PocMgrTab2},
+                [PocMgrTab1] = miner_poc_mgr:make_ets_table(),
+                POCMgrOpts = #{tab1 => PocMgrTab1},
```

Comment:
```suggestion
                [PocMgrTab] = miner_poc_mgr:make_ets_table(),
                POCMgrOpts = #{tab => PocMgrTab1},
```

Reply:
will make the change locally and push, you suggestion is still referencing `PocMgrTab1`